src/components/SearchForm.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="search-wrapper">
+    <div class="container">
+      <form
+        class="search-form"
+        @submit.prevent="onSubmit"
+      >
+        <img
+          class="search-icon"
+          src="../assets/search.svg"
+          alt="Search"
+          width="20"
+        >
+        <input
+          v-model.trim="searchValue"
+          class="search-input"
+          type="text"
+          placeholder="Search"
+        >
+        <label class="search-adult">  
```

Comment:
Please, take a look on bootstrap styles for checkboxes: https://getbootstrap.com/docs/5.2/forms/checks-radios/#checks